PIPS/pip-009.md

```diff
@@ -0,0 +1,58 @@
+    PIP: 009
+    Title: Investor Referral Reward Program (aka Hustler PIP)
+    Author: <KRostislav Grubi>
+    Created: <2018-03-18>
+    Requires: PIP002
+
+
+## Simple Summary
+Bring in new customers (investments) to the project and receive your cut as a reward. That can help us stay independent. 
+
+## Motivation
+Project needs money to sustain itself and stay independent. This PIP aims to incentivise people to look for a way to bring more customers (investments) to the PARSEC Labs project.
+
+## Specification
+
+**Glossary:**
+
+- Claimant - a person willing to claim reward as part of this PIP
+- Rewardable base - amount of ETH eligible for reward calculation
+
+**Things qualified for reward**
+
+1. You bring some customer (investor) which eventually invested in the project
+ 
+Rewardable base: ETH or fiat investment sent to Escrow council or our LLC by customer
```

Comment:
I would not mention any LLC here, if a LLC receives money, it can choose what to do with it, and cut itself a share before funding the escrow council. PIPs only concern management of funds once they are in the multisig already.